common/pkg/capi/clustertopology/handlers/mutation/meta.go

```diff
@@ -59,6 +68,18 @@ func (mgp metaGeneratePatches) GeneratePatches(
 	resp *runtimehooksv1.GeneratePatchesResponse,
 ) {
 	clusterKey := handlers.ClusterKeyFromReq(req)
+	getCluster := func(ctx context.Context) (*clusterv1.Cluster, error) {
```

Comment:
nit: match what is in the handlers?
```suggestion
	clusterGetter := func(ctx context.Context) (*clusterv1.Cluster, error) {
```